EWL-4913 checkbox atom

designerneil opened the pull request:
## Ticket(s)

**Jira Ticket**
- [EWL-4913: SG2 | Update checkbox atom](https://issues.ama-assn.org/browse/EWL-4913)

## Description
Adds checkbox atom

## To Test
- [ ] `gulp serve`
- [ ] visit http://localhost:3000/?p=atoms-checkbox
- [ ] Observe a checkbox with a label checkbo
- [ ] Click on the checkbox
- [ ] Observe the checkbox is purple with a white checkmark inside

## Visual Regressions

backstop test. No failures

## Relevant Screenshots/GIFs
<img width="162" alt="screen shot 2018-04-25 at 12 13 11 pm" src="https://user-images.githubusercontent.com/2271747/39261425-0aa42112-4882-11e8-9a1d-c85c70039ae0.png">


## Remaining Tasks
N/A


## Additional Notes
N/A
---

[Guidelines for Contribution](CONTRIBUTING.md)
[SG2 Standards](ama-style-guide-2/docs/standards.md)

designerneil committed:
Adds jquery ui checkbox

designerneil committed:
EWL-4913: Adds Checkbox atom

designerneil committed:
Merge branch 'develop' of github.com:AmericanMedicalAssociation/ama-style-guide-2 into feature/EWL-4913-checkbox-atom

# Conflicts:
#	styleguide/source/_patterns/01-atoms/forms/checkbox.md
#	styleguide/source/_patterns/01-atoms/forms/checkbox.twig
#	styleguide/source/assets/js/form-items.js

designerneil committed:
EWL-4913: Adds Checkbox atom

labbydev reviewed (commented):
@designerneil it looks like the default state should include a gray border with dark gray text, however I'm just seeing a gray drop shadow on one section with black text. 

**Pattern Lab**
<img width="134" alt="screen shot 2018-04-26 at 10 53 55 am" src="https://user-images.githubusercontent.com/1653723/39317101-660f36a4-4940-11e8-84b4-f141cf19bc23.png">

**Zeplin**
<img width="217" alt="screen shot 2018-04-26 at 10 54 58 am" src="https://user-images.githubusercontent.com/1653723/39317102-661b638e-4940-11e8-8d4a-9b26d412aac2.png">

designerneil commented:
@labbydev I want to go over the checkbox stuff with @toph in our UX collab meeting today. Will finalize the checkbox look and feel then.

designerneil committed:
Merge branch 'develop' of github.com:AmericanMedicalAssociation/ama-style-guide-2 into feature/EWL-4913-checkbox-atom

# Conflicts:
#	styleguide/source/assets/js/form-items.js

designerneil committed:
Address concerns made in PR
@designerneil it looks like the default state should include a gray border with dark gray text, however I'm just seeing a gray drop shadow on one section with black text.
